_posts/2016-05-08-release-0.15.0.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: Release 0.15.0
+---
+
+We have just released MDAnalysis version 0.15.0. This release contains new
+features as well as bug fixes. Highlights are listed below but for more details
+see the [release notes](https://github.com/MDanalysis/mdanalysis/wiki/...).
+
+We also had a lot of contributions from GSoC applicants. Thanks to our GSoC
+students @fiona-naughton and @jdetle as well as the other applicants @Saxenauts,
+@Endle, @abhinavgupta94 and @pedrishi.
+
+# Upgrade
+
+You can upgrade with `pip install --upgrade MDAnalysis`
+
+# Noticable Changes
+
+## Deprecations in anticipation of new topology system
+
+The next release (0.16.0) will bring a very big change to the internal workings of MDAnalysis.
+The topology system (how atom, residue, segment attributes are stored and manipulated) has been
+completely redesigned, giving many advantages and resolving a number of longstanding issues.
```

Comment:
Name or link a few, or the label on github. You should also mention that it will give a speed improvement and make it easier to maintain MDAnalysis in the future. 

The deprecations are an inconvenience for the users so lets sweeten the deal for them explicitly.